i18n/de.po

```diff
@@ -276,50 +280,30 @@ msgid "Website"
 msgstr "Webseite"
 
 #. module: website_sale_osc
-#: view:website:website_sale_osc.osc_onestepcheckout
-msgid "Your Company"
-msgstr "Firma"
-
-#. module: website_sale_osc
-#: view:website:website_sale_osc.osc_onestepcheckout
+#: view:website:website_sale_osc.address
 msgid "Your Name"
 msgstr "Ihr Name"
 
 #. module: website_sale_osc
-#: view:website:website_sale_osc.osc_onestepcheckout
-msgid "Your Order"
-msgstr "Ihr Auftrag"
+#: view:website:website_sale_osc.address
+msgid ""
```

Comment:
Not translated. I'm currently not know if this works (see above in Terms & Conditions)